public/css/style.css

```diff
@@ -1,5 +1,5 @@
 :root {
-    --brand-color: #013755;
+    --brand-color: #014055;
```

Comment:
Unfortunately the description is missing why this change is needed. Can you explain?

Thanks in advance!

Reply:
Changed the colour code, it should look better after changing it

Reply:
> it should look better after changing it

Maybe, but it doesn't matter. We are using a defined colour set in the whole project for logos, websites and so on. And this is one colour of the defined set.
You can find the same colour code also at the website of Laminas and the documentation: 

* https://getlaminas.org/
* https://docs.laminas.dev/

However, thank you for your interest in helping!